src/components/course/SubsidyRequestButton.jsx

```diff
@@ -89,7 +91,15 @@ const SubsidyRequestButton = () => {
     )
   );
 
-  if (!showSubsidyRequestButton) {
+  // Learner Credit B&R
+  const showLearnerCreditRequestButton = !showSubsidyRequestButton
+  && !userSubsidyApplicableToCourse
+  && !isUserEnrolled
+  && canRequestLearnerCredit
+  && learnerCreditRequestablePolicy
+  && !userHasSubsidyRequest;
```

Comment:
[suggestion] To make it more readable, this could be written as below. Also please drop a code comment explaining this logic.  
```javascript
const shouldHideLCRequestButton =
  showSubsidyRequestButton ||
  userSubsidyApplicableToCourse ||
  isUserEnrolled ||
  userHasSubsidyRequest;

const showLearnerCreditRequestButton =
  !shouldHideLCRequestButton &&
  canRequestLearnerCredit &&
  learnerCreditRequestablePolicy;